<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgriculturalMachinerCultivators</class>
 <widget class="QWidget" name="AgriculturalMachinerCultivators">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>490</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active/>
    <inactive/>
    <disabled/>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/new/prefix1/window/application_icon_.jpg</normaloff>:/new/prefix1/window/application_icon_.jpg</iconset>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>490</height>
    </rect>
   </property>
   <widget class="QSpinBox" name="processing_depth_lancet_paw_">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>205</y>
      <width>291</width>
      <height>190</height>
     </rect>
    </property>
    <property name="title">
     <string>Результаты расчета</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Общее тяговое сопротивление, Н</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Необходимая мощность </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Тяговвая категория, по ГОСТ</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_result_resistance_lancet_paw_">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Сопротивления одной лапы, Н</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_result_resistance_rink_">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Сопротивления одного диска, Н</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>тягового агрегата, л.с.</string>
     </property>
    </widget>
    <widget class="QLabel" name="result_traction_resistance_">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>170</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="result_power_tractor_">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>170</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="result_resistance_lancet_paw_">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>170</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="result_traction_category_">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>130</y>
       <width>170</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="result_resistance_rink_">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>170</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>-280</x>
       <y>0</y>
       <width>600</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Необходимая мощность</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>130</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>тягового агрегата, кВт</string>
     </property>
    </widget>
    <widget class="QLabel" name="result_power_kW_">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>170</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="soil_type_">
    <property name="geometry">
     <rect>
      <x>371</x>
      <y>158</y>
      <width>140</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>лег. / сред. суглинки</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>тяжелые суглинки</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>сухие глинистые</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Глубина обработки,мм</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Общий вес агрегата, кг</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_Disc_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Диск лущильника</string>
    </property>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>85</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Диаметр D, мм</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Количество</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="diametr_rink_">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>60</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>на агрегат, шт.</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="quantity_disc_">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="attack_angle_">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>85</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Угол атаки, f&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_paw_width_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>271</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Стрельчастая лапа</string>
    </property>
    <widget class="QLineEdit" name="lancet_paw_width_">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>60</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ширина L,мм</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Количество</string>
     </property>
    </widget>
    <widget class="QLabel" name="scheme_lancet_paw_">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>15</y>
       <width>141</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>на агрегат, шт</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="quantity_paw_width_">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="estimation_">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>415</y>
      <width>112</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Произвести расчет</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="total_weight_">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип почвы</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="speed_">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Рабочая скорость, км/ч.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_fields_">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>415</y>
      <width>112</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить поля</string>
    </property>
   </widget>
   <widget class="QLabel" name="botton_hand_">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>600</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Ошибка! Данные введены неверно.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QToolButton" name="back_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>39</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>35</y>
      <width>40</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QLabel" name="upper_band_">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ширина захвата, м</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="grip_width_">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="collapse_">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>5</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="close_">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>5</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="name_program_">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>EMAM</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="availability_of_ice_rinks_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>435</y>
      <width>220</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Наличие задних катков на агрегате</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
